dev-python/python-lsp-jsonrpc/python-lsp-jsonrpc-1.0.0.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2021 Gentoo Authors
```

Comment:
How do these forks work? Does spyder require them in the new version unconditionally, or prefers over the original packages, or...?

Reply:
These forks are the continued versions of python-language-server/python-jsonrpc-server the original versions are no longer actively developed. Spyder currently does not require them yet, but it will in the next release, and once that version is released we can remove the original package and add an move entry to `profiles/updates`

Reply:
No, you won't be able to add a move entry once a new package is added. But if they install under a different name (i.e. not collide with the old upstream), then a move entry is not the right thing to do anyway.

Reply:
Why not? It's the same thing just distributed under a different name. We would want users who have `dev-python/python-language-server` installed to automatically migrate to `dev-python/python-lsp-server` right? A move entry would accomplish that, or am I mistaken?

Reply:
No, move entry would push two packages into the same CAT-PN-SLOT, resulting in invalid state.

Reply:
Hmm, I see, so how about we add the new package, stabilize the new package, then mask the old package (with a message pointing to the new package), and then remove the old package later. That should work right?

Reply:
Yes, I think that's the best option.